sources/main.functions.php

```diff
@@ -1932,9 +1929,9 @@ function defuseFileEncrypt(
     } catch (CryptoException\WrongKeyOrModifiedCiphertextException $ex) {
         $err = 'wrong_key';
     } catch (CryptoException\EnvironmentIsBrokenException $ex) {
-        $err = $ex;
+        $err = print_r($ex, true);
     } catch (CryptoException\IOException $ex) {
-        $err = $ex;
+        $err = print_r($ex, true);
```

Comment:
Does this can leak server private informations to frontend (server saltkey, saltkey path or other)?

Reply:
I have improved this.

Thanks